hub/auto/tests/test_tabular_data.py

```diff
@@ -0,0 +1,59 @@
+import json
+import os
+import shutil
+import zipfile
+from pathlib import PosixPath
+
+import hub
+import numpy as np
+import pytest
+from hub.auto.tests.util import get_dataset_store
+from hub.utils import pandas_loaded
+
+
+def assert_conversion(tag, dataset_shape, max_review_shape, max_filename_shape):
+
+    dataset_store = get_dataset_store(tag)
+    hub_dir = dataset_store / "hub"
+
+    # delete hub dataset so conversion test can be done
+    if hub_dir.is_dir():
+        print("hub_dir was found (%s), deleting..." % hub_dir)
+        shutil.rmtree(hub_dir)
+
+    try:
+        ds = hub.Dataset.from_path(str(dataset_store))
+    except Exception:
+        assert False
+
+    print("dataset obj:", ds)
+    assert ds is not None
+
+    assert hub_dir.is_dir(), hub_dir
+
+    if dataset_shape is not None:
+        assert dataset_shape == ds.shape
+
+    if max_review_shape is not None:
+        actual_max_review_shape = np.max(ds["Review"].shape)
```

Comment:
i modified the `dummy_data` csv files, so these tests no longer make sense. please rewrite these tests to pass.

## please do the following:
1. make sure that the hub dataset has correctly parsed **every single value** (use pandas dataframe as your baseline to compare). this shouldn't be hard & will make it more general so you can add any csv files here, rather than hard coded
2. make sure the hub dataset has correctly parsed the datatypes. if the pandas dataframe column sees it as an integer, but it is loaded as a float: this is a failure case.
3. you should also check to see if the values are being stored correctly. don't just check the filename length, you should check the filename & see if it is exactly correct.
4. please make the csv files a little bit more diverse. i just quickly made them, but you should add as many datatypes & cases you can think of. i know it isn't fun, but we need to be very precise here.